tests/integ/test_stored_procedure.py

```diff
@@ -1497,3 +1497,28 @@ def f(session: Session) -> int:
             f, packages=["snowflake-snowpark-python"], force_inline_code=True
         )
     assert any("AS $$" in query.sql_text for query in query_history.queries)
+
+
+@pytest.mark.skipif(
+    not is_pandas_available, reason="test module dependency with pandas dependency"
+)
+def test_stored_proc_register_with_module(session):
+
+    import pandas
+
+    # use pandas module here
+    packages = list(session.get_packages().values())
+    assert "pandas" "pandas" not in packages
```

Comment:
Is `"pandas" "pandas"` intended?

Reply:
If it's part of the error message may be put that in a variable that is referenced to avoid any confusion?

Reply:
good catch, this is a typo. fixed.